### ISSUE TITLE
a11y_NativeSelect> Has error> A11y> 1 violations is appearing

### ISSUE THREAD

**Issue by priya-rajput** [cengage/react-magma#502]
**Describe the bug**
NativeSelect>> "1 violations" is appearing for "Has error". 

**To Reproduce**
1. Go to 'https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app/?path=/story/nativeselect--has-error'
2. Click on 'Accessibility" add on
3. "1 violations" is appearing

**Expected behavior**
"0 violations" should appear for NativeSelect> Has error.

**Screenshots**
https://somup.com/c3n2j7ZW8W

**Desktop:**
 - OS: [Win 10]
 - Browser [chrome]
 - Version [Version 98.0.4758.82 (Official Build) (64-bit)]

**Comment by Uditsingh67** [comment]
1 violations is still appearing .Violations does not come if user reloads the page.
See Screen-cast : http://somup.com/c3jffHuCCJ
Thanks!

**Comment by ccedrone** [comment]
This appears to be an issue with Storybook and not the code itself. Leslie Schulz verified it as a11y compliant. No plans to fix.
